### ISSUE TITLE
Report

### ISSUE THREAD

**Issue by F0h** [nextcloud/android#7720]
************ CAUSE OF ERROR ************

java.lang.NoClassDefFoundError: Failed resolution of: Lokio/Buffer;
	at okhttp3.ResponseBody.create(ResponseBody.java:210)
	at okhttp3.internal.Util.<clinit>(Util.java:62)
	at okhttp3.Credentials.basic(Credentials.java:30)
	at com.owncloud.android.lib.common.OwnCloudClientFactory.createNextcloudClient(OwnCloudClientFactory.java:217)
	at com.owncloud.android.operations.RefreshFolderOperation.updatePredefinedStatus(RefreshFolderOperation.java:330)
	at com.owncloud.android.operations.RefreshFolderOperation.updateCapabilities(RefreshFolderOperation.java:304)
	at com.owncloud.android.operations.RefreshFolderOperation.updateOCVersion(RefreshFolderOperation.java:276)
	at com.owncloud.android.operations.RefreshFolderOperation.run(RefreshFolderOperation.java:229)
	at com.owncloud.android.lib.common.operations.RemoteOperation.run(RemoteOperation.java:360)
	at java.lang.Thread.run(Thread.java:818)
Caused by: java.lang.ClassNotFoundException: Didn't find class "okio.Buffer" on path: DexPathList[[zip file "/data/app/com.nextcloud.android.beta-2/base.apk"],nativeLibraryDirectories=[/data/app/com.nextcloud.android.beta-2/lib/arm, /vendor/lib, /system/lib]]
	at dalvik.system.BaseDexClassLoader.findClass(BaseDexClassLoader.java:56)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:511)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:469)
	... 10 more
	Suppressed: java.lang.ClassNotFoundException: okio.Buffer
		at java.lang.Class.classForName(Native Method)
		at java.lang.BootClassLoader.findClass(ClassLoader.java:781)
		at java.lang.BootClassLoader.loadClass(ClassLoader.java:841)
		at java.lang.ClassLoader.loadClass(ClassLoader.java:504)
		... 11 more
	Caused by: java.lang.NoClassDefFoundError: Class not found using the boot class loader; no stack available

************ APP INFORMATION ************
ID: com.nextcloud.android.beta
Version: 20201216
Build flavor: versionDev

************ DEVICE INFORMATION ************
Brand: alps
Device: elink8321_emmc
Model: SUPERNOVA I3G
Id: LMY47I
Product: full_elink8321_emmc

************ FIRMWARE ************
SDK: 22
Release: 5.1
Incremental: 1524737742

### Steps to reproduce
1. 
2. 
3. 

### Expected behaviour
- Tell us what should happen

### Actual behaviour
- Tell us what happens

### Can you reproduce this problem on https://try.nextcloud.com?
- Please create a test demo account and see if this still happens there.
- If yes, please open up a bug report
- If not, please verify server setup and ask for help on forum

### Environment data
Android version:

Device model: 

Stock or customized system:

Nextcloud app version:

Nextcloud server version:

Reverse proxy:

#### Web server error log
```
Insert your webserver log here
```

#### Nextcloud log (data/nextcloud.log)
```
Insert your Nextcloud log here
```
**NOTE:** Be super sure to remove sensitive data like passwords, note that everybody can look here! You can use the Issue Template application to prefill some of the required information: https://apps.nextcloud.com/apps/issuetemplate

**Comment by KwonEunKyung** [comment]
What procedure is the error? I think that part is missing.

**Comment by AndyScherzinger** [comment]
This has been fixed and 8s available with the latest Dev build 20201223, please Update 😊
